### PR TITLE
NewYorkTimes-Article-Page

### PR DESCRIPTION
**Pull request by elasfarc**
In this project we worked on the following:
	•	Created HTML body & CSS styles.
	•.     Create a fixed Navbar
	•.     Added layouts in Grid, Flex, position and Float
	•.     Added hovers to actionable links.
	•.     Added relevant images, videos, links and icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,876 @@
...
+                                      615
+                                    </sub>
+                                  </span>
+
+                               </a>
+                            </li>
+                        </ul>
+
+
+                     <!--  <i class="far fa-comment-alt " id="icon-box" ><span id="icon-num" ></span></i>-->
```
Comment (peterrobert) [inline index.html]: kindly remove commented code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,876 @@
...
+
+        <div class="inflationBOX" >
+            <div class="inflation-head">
+                <h3>The Theory of Inflation</h3>
+                <span>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.
+                </span>
+            </div>
+            <div class="inflationDIV" >
+             <!--<img id="img-in" src="./assets/inflation-720.png">-->
+               <div class="row1p">
```
Comment (peterrobert) [inline index.html]: kindly remove commented code it keeps your work clean

File: assets/stylesheet/styles.css [hunk 3]
```diff
@@ -0,0 +1,824 @@
...
+}
+
+/* YT figure */
+
+#YT-video {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
```
Comment (peterrobert) [inline assets/stylesheet/styles.css:789]: There are a few CSS errors at this point kindly fix them.
warnings css

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
### HELLO @elasfarc AND @Fanger53  :wave:  AM PETER ROBERT A TSE AND I HAVE REVIED YOUR PROJECT.
### YOUR PROJECT LOOKS AMAZING FOR A FIRST REVIEW. IT ONLY HAVE SOME FEW CHANGES TO BE MADE. KINDLY CHCKE THE CHANGES BELOW :arrow_down: 

- [x] Kindly add a space logo to your project. Just like in the original :arrow_down: 
### original
space logo

- [x] Kindly you need to add a fixed footer to your project just like in the original :arrow_down: 
### original
fixed footer

### [OPTIONAL]
Your HTML has a lot of warnings. Since they are not errors it is not a requirement to fix them.
warnings

When you are done kindly submit your project for another review.

### if you have a question concerning the review kindly reach me via slack on @peterrobert

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :green_circle: 

Your clone looks awesome :clap: 

However, you still have a commented code on line 243. Since you checked all the minimum requirements and removed other ones that the previous reviewer pointed out, I'm not going to reprove it because I'm assuming that you missed this one. Kindly remove it because it is not a good practice.

You did great on this :100: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**